src/setting/edit_setting.js

```diff
@@ -0,0 +1,40 @@
+document.querySelector('#btn-up').addEventListener('click', () => {
+
+    var s = 
+`[default]
+client_name     = '${document.getElementById('username').value}'
+client_key      = '${document.getElementById('certificate').value}'
+chef_server_url = '${document.getElementById('url').value}'\n`
+
+window.obj = {
+    "client_name     ": document.getElementById('username').value,
+    "client_key      ": document.getElementById('certificate').value,
+    "chef_server_url ": document.getElementById('url').value
+}
+
+const fs = require('fs');
+
+try {
+    fs.writeFileSync("/Users/ngupta/chef-repo/.chef/credentials", s);
```

Comment:
This filename should be generic, something like` '$HOME/.chef/credentials'`

Reply:
this needs to be updated, once the design is ready based on that this will be updated